github/ui/styles.css

```
@@ -0,0 +1,496 @@
+  color: var(--color-ink-faint);
+  text-transform: lowercase;
+  font-weight: 600;
+  border-bottom: 1px solid var(--color-rule);
+}
+[data-iii-ui="github"] .gh-rv-num {
+  color: var(--color-ink-faint);
+  white-space: nowrap;
+}
+[data-iii-ui="github"] .gh-rv-title {
+  color: var(--color-ink);
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)